legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -355,6 +344,28 @@ class FilingTitles(str, Enum):
                 'CC': 'RESXF'
             }
         }
+    },
+    'specialResolution': {
+        'name': 'specialResolution',
+        'title': 'Special Resolution',
+        'displayName': 'Special Resolution',
+        'codes': {
+            'CP': 'SPRLN'
+        }
+    },
+    'transition': {
+        'name': 'transition',
+        'title': 'Transition',
+        'displayName': 'Transition Application',
+        'codes': {
+            'BC': 'TRANS',
+            'BEN': 'TRANS',
+            'ULC': 'TRANS',
+            'CC': 'TRANS'
+        },
+        'additional': [
+            {'types': 'BC,BEN', 'outputs': ['noticeOfArticles', ]},
+        ]
```

Comment:
Is there a reason to put these in alphabetical order, other than style guides/consistency?

Reply:
nope